backend/frontend/gulpfile.js

```diff
@@ -53,7 +53,7 @@ function build_node(){
 
   return gulp.src("src/**/*.js")
       .pipe(sourcemaps.init())
-      .pipe(babel({ optional: ['runtime'] }))
+      // .pipe(babel({ optional: ['runtime'] }))
```

Comment:
If we're not using them, let's just delete the lines if that makes sense. Also, will this remove babel from our package dependencies? Any dependencies we can safely remove, we probably should do so.